### PR TITLE
chore: make sprout optional

### PR DESCRIPTION
**Pull request by Caele**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -43,6 +45,7 @@ export default function supernova(env) {
       useEffect(() => {
         if (layout && tableData) {
           render(el, { tableData, setPageInfo, pageInfo, constraints, selections, muiParameters });
+          render(el, { tableData, setPageInfo, pageInfo, constraints, selections });
```
Comment (haxxmaxx) [inline src/index.js:48]: Should there be two render calls here?

File: src/index.js [hunk 2]
```diff
@@ -6,6 +6,8 @@ import ext from './ext';
 import muiSetup from './mui-setup';
 import { render, teardown } from './table/root';
 import manageData from './table/handle-data';
+// This line is replaced by rollback with an import for internal builds
```
Comment (haxxmaxx) [inline src/index.js:9]: rollup?

File: src/index.js [hunk 3]
```diff
@@ -6,6 +6,8 @@ import ext from './ext';
...
+const __OPIONAL_THEME_DEPS__ = {}; // eslint-disable-line no-underscore-dangle
```
Comment (haxxmaxx) [inline src/index.js:10]: I don't know why you use __ before/after the variable name, does it mean it has to do with building or something?

File: src/index.js (hunk 1 above)
Comment (Caele) [reply src/index.js:48]: double the fun?
No, merge problem

File: src/index.js (hunk 2 above)
Comment (Caele) [reply src/index.js:9]: yes

File: src/index.js (hunk 3 above)
Comment (Caele) [reply src/index.js:10]: yea, more or less like a warning "DON*T TOUCH!".

**Review by haxxmaxx — APPROVED** [review]

**Pull request merged**